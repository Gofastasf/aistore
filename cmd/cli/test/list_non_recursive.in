ais archive gen-shards "ais://$BUCKET_1/shard-{0..2}.tar" --fcount 1 --fsize 1KB --cleanup // IGNORE
ais archive gen-shards "ais://$BUCKET_2/subdir/shard-{0..2}.tar" --fcount 1 --fsize 1KB --cleanup // IGNORE

# Add plain content with simple nesting
ais put /dev/null ais://$BUCKET_1/file1.txt --yes // IGNORE
ais put /dev/null ais://$BUCKET_1/docs/readme.txt --yes // IGNORE
ais put /dev/null ais://$BUCKET_1/docs/nested/deep.txt --yes // IGNORE

ais ls --nr ais://$BUCKET_1
ais ls --nr ais://$BUCKET_2

# Prefix tests - with trailing "/"
ais ls --nr ais://$BUCKET_1/shard-
ais ls --nr ais://$BUCKET_1/docs/
ais ls --nr ais://$BUCKET_1/docs/nested/



ais bucket rm ais://$BUCKET_1 // IGNORE
ais bucket rm ais://$BUCKET_2 // IGNORE
