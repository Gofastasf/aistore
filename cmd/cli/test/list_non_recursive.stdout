NAME             SIZE            
docs/                    
file1.txt        0B              
shard-0.tar      2.50KiB         
shard-1.tar      2.50KiB         
shard-2.tar      2.50KiB
NAME             SIZE
subdir/
NAME             SIZE            
shard-0.tar      2.50KiB         
shard-1.tar      2.50KiB         
shard-2.tar      2.50KiB
NAME             SIZE    
docs/            
docs/readme.txt  0B
NAME                     SIZE    
docs/nested/             
docs/nested/deep.txt     0B